jetty-websocket/jetty-websocket-tests/src/test/java/org/eclipse/jetty/websocket/tests/client/ClientCloseTest.java

```diff
@@ -83,10 +83,7 @@ private Session confirmConnection(CloseTrackingEndpoint clientSocket, Future<Ses
         {
             // Send message from client to server
             final String echoMsg = "echo-test";
-            Future<Void> testFut = clientSocket.getRemote().sendStringByFuture(echoMsg);
-
-            // Wait for send future
-            testFut.get(5, SECONDS);
+            clientSocket.getRemote().sendString(echoMsg);
```

Comment:
Should this method still return Session?

Reply:
I think `confirmConnection()` should still return a session because it is given the `Future<Session>` and is just returning the result. But the result is now unused so it could return void.